tests/drivers/spi/spi_loopback/boards/nucleo_g431rb.conf

```diff
@@ -0,0 +1,12 @@
+CONFIG_SPI_LOOPBACK_DRV_NAME="SPI_1"
+
+CONFIG_SPI_STM32_DMA=y
+CONFIG_SPI_STM32_INTERRUPT=n
+
+CONFIG_SPI_ASYNC=y
+
+CONFIG_SPI_LOOPBACK_MODE_LOOP=n
+
+# Reduce bus clock speed to be able to reach SPI_LOOPBACK_SLOW_FREQ = 500000
+# with max prescaler 256
+CONFIG_CLOCK_STM32_APB2_PRESCALER=2
```

Comment:
Please convert to device tree.

Reply:
Thanks for pointing this out.
I see that the pr that changes the config to dt was not merged for nucleo_g474re.
It would probably be best to update it for nucleo_g474re as well, and block until #34709 was merged?

Not related to this pr, but there is a similar case in watchdog/wdt_basic_api, have you already taken account for that one?

Reply:
> Thanks for pointing this out.
> I see that the pr that changes the config to dt was not merged for nucleo_g474re.
> It would probably be best to update it for nucleo_g474re here as well, and block until #34709 was merged, or would you prefer that #34709 accounts for this itself?

Aim is to get https://github.com/zephyrproject-rtos/zephyr/pull/34709 merged today.

> Not related to this pr, but there is a similar case in watchdog/wdt_basic_api, have you already taken account for that one?

Sure, I'll take care about it.